include/aktualizr-lite/tuf/tuf.h

```diff
@@ -197,6 +197,7 @@ class Repo {
   Repo& operator=(const Repo&&) = delete;
 
   virtual std::vector<TufTarget> GetTargets() = 0;
+  virtual std::string GetLatestRoot() = 0;
```

Comment:
Can we name it as `getRoot(int version)`? If `version` equals to `-1` then return the latest version.

Reply:
Have you considered making it `GetRoot(int version)` so it cover all use-cases?

Reply:
Yes, it LGTM, please go ahead, test, and merge it.